### PR TITLE
Merging PS FAQ into individual docs #2

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/265338182

Changes proposed in this pull request:

- Move FAQs into individual docs
- staged:
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/faq/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fedcm/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fenced-frame/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/first-party-sets/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/topics/topic-classification/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/trust-tokens/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/unified-origin-trial/
    - https://pr-5654-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/user-agent/
- plus: [site/en/_partials/privacy-sandbox/topics-feedback.md](https://github.com/GoogleChrome/developer.chrome.com/pull/5654/files#diff-e26fa8c27a54c9fedc74da660fda2079c4b0043d780db6d9d273dbbfd45807b2)

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/index.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ If you're unfamiliar with some of these terms, consult the
 
 Keep track of the [API changes](/docs/privacy-sandbox/attribution-reporting-updates/).
 
+{% Aside 'note' %}
+Attribution Reporting is the new name for the Event Conversion Measurement API. [The name was changed](/docs/privacy-sandbox/attribution-reporting-introduction/), as the original event-level scope expanded to cover additional measurement use cases.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/attribution-reporting/index.md:38]:
```suggestion
Attribution Reporting was formerly known as the Event Conversion Measurement API. [The name was changed](/docs/privacy-sandbox/attribution-reporting-introduction/) in 2022, as the original event-level scope expanded to cover additional measurement use cases.
```

It's not really "new" anymore, as it's been over a year. However, it's still worth the note, hence the rewrite.

File: site/en/docs/privacy-sandbox/fenced-frame/index.md [hunk 2]
```diff
@@ -185,6 +222,9 @@ Current candidates for this combination include:
    or [access unpartitioned
    storage](https://github.com/shivanigithub/fenced-frame/blob/master/explainer/modes.md#unpartitioned-storage).
 
+For ads use cases, see
+[Fenced frames for Ads design document](https://docs.google.com/document/d/17rtX55WkxMcfh6ipuhP4mNULIVxUApvYt4ZYXfX2x-s/edit#heading=h.jy0hectpkl95).
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/fenced-frame/index.md:226]:
```suggestion
```

The linked doc is obsolete, so let's remove the reference.

**Review by heyawhite — COMMENTED** [review]
A few suggestions. There are some web.dev links that have content that is now on DCC. In particular, how to participate in an origin trial: https://developer.chrome.com/docs/web-platform/origin-trials/

I think I've caught all references.

I want to be sure that our partnerships team can review the User-Agent changes, and then will approve!

**Review by heyawhite — APPROVED** [review]

File: site/en/_partials/privacy-sandbox/topics-feedback.md [hunk 3]
```diff
@@ -3,4 +3,10 @@
 - **GitHub**: Read the Topics API [explainer](https://github.com/jkarlin/topics), and [raise questions and follow discussion in issues on the proposal repo](https://github.com/jkarlin/topics/issues).
 - **W3C**: Discuss industry use cases in the [Improving Web Advertising Business Group](https://www.w3.org/community/web-adv/participants).
 - **Announcements**: [Join or view the mailing list](http://groups.google.com/a/chromium.org/g/topics-api-announce).
-- **Privacy Sandbox developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
\ No newline at end of file
+- **Privacy Sandbox developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
+- For questions about the proposal: 
+[create an issue](https://github.com/jkarlin/topics/issues) on the proposal repo.
```
Comment (heyawhite) [inline site/en/_partials/privacy-sandbox/topics-feedback.md:8]:
```suggestion
```

This is the same content under **GitHub**

File: site/en/_partials/privacy-sandbox/topics-feedback.md [hunk 4]
```diff
@@ -3,4 +3,8 @@
...
 - **W3C**: Discuss industry use cases in the [Improving Web Advertising Business Group](https://www.w3.org/community/web-adv/participants).
 - **Announcements**: [Join or view the mailing list](http://groups.google.com/a/chromium.org/g/topics-api-announce).
-- **Privacy Sandbox developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
\ No newline at end of file
+- **Privacy Sandbox developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
+- For questions about the implementation currently available to test in 
+Chrome: [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=topics).
+- For implementation, integration, and general best practice questions: 
+[create an issue](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support) on the Privacy Sandbox developer support repo.
```
Comment (heyawhite) [inline site/en/_partials/privacy-sandbox/topics-feedback.md:10]:
```suggestion
```

This is the same resources as "Privacy Sandbox developer support"

File: site/en/_partials/privacy-sandbox/topics-feedback.md [hunk 5]
```diff
@@ -3,4 +3,6 @@
 - **GitHub**: Read the Topics API [explainer](https://github.com/jkarlin/topics), and [raise questions and follow discussion in issues on the proposal repo](https://github.com/jkarlin/topics/issues).
 - **W3C**: Discuss industry use cases in the [Improving Web Advertising Business Group](https://www.w3.org/community/web-adv/participants).
 - **Announcements**: [Join or view the mailing list](http://groups.google.com/a/chromium.org/g/topics-api-announce).
-- **Privacy Sandbox developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
\ No newline at end of file
+- **Privacy Sandbox developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
+- For questions about the implementation currently available to test in 
+Chrome: [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=topics).
```
Comment (heyawhite) [inline site/en/_partials/privacy-sandbox/topics-feedback.md:8]:
```suggestion
- **Chromium**: [File a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=topics) to ask questions about the implementation currently available to test in Chrome.
```

Makes this parallel in style to the other bullets.

**Review by heyawhite — APPROVED** [review]
Final nit pre-merge

**Comment by nancymic2** [comment]
> Final nit pre-merge

I'm not seeing this?

**Comment by heyawhite** [comment]
@nancymic2 no action needed, I've merged that feedback. Merging the FAQs now.

**Pull request merged**